source/blog/2018-03-06-the-many-ways-to-build-oci-images.md

```diff
@@ -0,0 +1,401 @@
+---
+title: The Many Ways to Build an OCI Image without Docker
+author: miabbott
+date: 2018-03-06 00:00:00 UTC
+published: true
+comments: false
+tags:
+- atomic
+- buildah
+- containers
+- docker
+- oci
+- ostree
+- runc
+- skopeo
+---
+
+When containers initially made their big splash into the industry via Docker,
+users were almost required to use the `docker` CLI and daemon to create and
+manage their container images.  But a lot has happened since then and now it
+is easier than ever to create a container image without using `docker` at all.
```

Comment:
Maybe also mention in here that the Docker format is now [standardized as OCI](https://github.com/opencontainers/image-spec)?

Reply:
Can do!